src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -2510,20 +2564,69 @@ public void Inheritance_07()
     public abstract int X { get; }
     public virtual int Y { get; }
 }
-data class B(int X, int Y) : A
+abstract data class B1(int X, int Y) : A
+{
+}
+data class B2(int X, int Y) : A
 {
 }";
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics(
-                // (6,12): error CS0534: 'B' does not implement inherited abstract member 'A.X.get'
-                // data class B(int X, int Y) : A
-                Diagnostic(ErrorCode.ERR_UnimplementedAbstractMethod, "B").WithArguments("B", "A.X.get").WithLocation(6, 12));
+                // (9,12): error CS0534: 'B2' does not implement inherited abstract member 'A.X.get'
+                // data class B2(int X, int Y) : A
+                Diagnostic(ErrorCode.ERR_UnimplementedAbstractMethod, "B2").WithArguments("B2", "A.X.get").WithLocation(9, 12));
+
+            AssertEx.Equal(new string[0], GetProperties(comp, "B1").ToTestDisplayStrings());
+            AssertEx.Equal(new string[0], GetProperties(comp, "B2").ToTestDisplayStrings());
         }
 
         [Fact]
         public void Inheritance_08()
         {
             var source =
+@"abstract class A
+{
+    public abstract int X { get; }
+    public virtual int Y { get; }
```

Comment:
Please add a test like this (`X`) but in which the type of the property in `A` and in `C` are incompatible. There should be an error that the override has the wrong type.

Reply:
Or do it when handling [https://github.com/dotnet/roslyn/issues/44785](https://github.com/dotnet/roslyn/issues/44785)

---
In reply to: [434231286](https://github.com/dotnet/roslyn/pull/44705#discussion_r434231286) [](ancestors = 434231286)

Reply:
Added https://github.com/dotnet/roslyn/issues/44785#issuecomment-637872297